.babelrc

```diff
@@ -1,5 +1,5 @@
 {
   "presets": [
-    "es2015"
+    "es2015-native-modules"
```

Comment:
They seem to be equal.

I looked at the source code and it seems like the `-webpack` is better.

* It removes the plugin from the preset by name, `-native-modules` removes it by index (less error prone)
* It is tested on travis

So we better change it to `-webpack`...